src/example/GeneSpy.jsx

```diff
@@ -0,0 +1,184 @@
+import React, {useCallback, useMemo, useState} from 'react';
+import {GoslingComponent} from 'gosling.js';
+import {Vega} from 'react-vega'
+
+const width = 300;
+const height = 300;
+
+function GeneSpy() {
+	// track order to link leaves of vega tree to tracks in gosling
+	const [trackOrder, setTrackOrder]=useState([])
+	const [maxDist,setMaxDist]=useState(1)
+	const onNewView = useCallback(view=>{
+		let leaves=view.data('leaves').slice();
+		leaves.sort((a,b)=>a.x-b.x);
+		setTrackOrder(leaves.map(leaf=>leaf.id))
+		setMaxDist(Math.max(...leaves.map(d=>d.distance)))
+	},[])
+	const goslingSpec=useMemo(()=>{
+		return({
+			title: 'GeneSpy + iTol',
+			subtitle: 'Genomic neighborhood visualization with aligned phylogenetic tree',
+			description: 'Idea: https://doi.org/10.1093/bioinformatics/bty459',
+			assembly: [['', 11000]],
+			static: true,
+			views: [
+				{
+					alignment: 'overlay',
+					data: {
+						url: 'https://s3.amazonaws.com/gosling-lang.org/data/GeneSpy/gene_spy_example.csv',
+						type: 'csv',
+						genomicFields: ['Gene start', 'Gene end']
+					},
+					dataTransform: [
+						{type: 'genomicLength', startField: 'Gene start', endField: 'Gene end', newField:'Gene length'}
+					],
+					color: {
+						field: 'type',
+						type: 'nominal',
+						domain: ['anchor', 'conserved', 'disrupted'],
+						range: ['red', 'gray', 'yellow']
+					},
+					row: {field: 'Accession', type: 'nominal',domain: trackOrder},
+					tracks: [
+						{
+							dataTransform: [
+								{type: 'filter', field: 'Strand', oneOf: ['+']}
+							],
+							mark: 'triangleRight',
+							style: {align: 'right'},
+							size: {field:'Gene length'},
```

Comment:
Do these actually have an effect on the vis when x and xe are both used?

Reply:
No, it doesn't I was experimenting here and forgot to remove this. Thanks for spotting this!